src/pages/saksbehandling/revurdering/endringAvFradrag/EndringAvFradrag.tsx

```diff
@@ -59,72 +65,97 @@ const EndringAvFradrag = (props: { sakId: string; revurdering: Revurdering }) =>
+                        tilOgMed: props.revurdering.periode.tilOgMed,
+                    },
+                    fradrag: values.fradrag.map((f: FradragFormData) => ({
+                        periode: null,
+                        /* valideringa sjekker at f.beløp og f.type ikke er null */
+                        /* eslint-disable @typescript-eslint/no-non-null-assertion */
+                        beløp: Number.parseInt(f.beløp!, 10),
+                        type: f.type!,
+                        utenlandskInntekt: f.fraUtland
+                            ? {
+                                  beløpIUtenlandskValuta: Number.parseInt(f.utenlandskInntekt.beløpIUtenlandskValuta),
+                                  valuta: f.utenlandskInntekt.valuta,
+                                  kurs: Number.parseFloat(f.utenlandskInntekt.kurs),
+                              }
+                            : null,
+                        tilhører: f.tilhørerEPS ? FradragTilhører.EPS : FradragTilhører.Bruker,
+                    })),
+                })
+            )
         );
+    };
 
     const schema = erGregulering(props.revurdering.årsak)
         ? yup.object<EndringAvFradragFormData>({
               fradrag: yup.array(fradragSchema.required()).defined(),
+              uføregrad: (yup
+                  .number()
+                  .nullable()
+                  .defined()
+                  .when('status', {
+                      is: UføreResultat.VilkårOppfylt,
+                      then: yup.number().positive().min(1).max(100).required().typeError('Feltet må være et tall'),
+                      otherwise: yup.number().nullable().defined(),
+                  }) as unknown) as yup.Schema<string>,
               forventetInntekt: (yup
                   .number()
-                  .typeError('Forventet inntekt etter uførhet må være et tall')
-                  .label('Forventet inntekt etter uførhet')
-                  .nullable(false)
-                  .integer()
-                  .min(0) as yup.Schema<unknown>) as yup.Schema<Nullable<string>>,
+                  .nullable()
+                  .defined()
+                  .when('status', {
+                      is: UføreResultat.VilkårOppfylt,
+                      then: yup.number().positive().integer().min(0).required().typeError('Feltet må være et tall'),
+                      otherwise: yup.number().nullable().defined(),
+                  }) as unknown) as yup.Schema<string>,
           })
         : yup.object<EndringAvFradragFormData>({
               fradrag: yup.array(fradragSchema.required()).defined(),
               forventetInntekt: yup.string().nullable().defined(),
+              uføregrad: yup.string().nullable().defined(),
           });
-    const forventetInntekt = props.revurdering.behandlingsinformasjon.uførhet?.forventetInntekt;
+    const uføregrunnlag = props.revurdering.grunnlag.uføre[0];
```

Comment:
`uføre[0]`? 😬

Reply:
Ja, vi har blittt revet litt mellom forskjellige modeller. Så det kommer til å være en liste rimelig snart, men akkurat nå garanterer backenden for et element, så lenge forrige vedtak var innvilget. Det du ser her er allerede deprecated. Ingar sine endringer gjør om på dette.